lib/msal-common/src/server/ServerTokenRequestParameters.ts

```diff
@@ -5,25 +5,28 @@
 import { ICrypto } from "../crypto/ICrypto";
 import { ScopeSet } from "../auth/ScopeSet";
 import { TokenExchangeParameters } from "../request/TokenExchangeParameters";
-import { ProtocolUtils } from "../utils/ProtocolUtils";
 import { Constants, HEADER_NAMES, AADServerParamKeys } from "../utils/Constants";
 import { ServerRequestParameters } from "./ServerRequestParameters";
 import { CodeResponse } from "../response/CodeResponse";
+import { Account } from "../auth/Account";
+import { StringUtils } from "../utils/StringUtils";
+import { ClientAuthError } from "../error/ClientAuthError";
 
 export class ServerTokenRequestParameters extends ServerRequestParameters {
 
     // Params
     tokenRequest: TokenExchangeParameters;
     codeResponse: CodeResponse;
+    refreshToken: string;
 
-    constructor(clientId: string, tokenRequest: TokenExchangeParameters, codeResponse: CodeResponse, redirectUri: string, cryptoImpl: ICrypto) {
+    constructor(clientId: string, tokenRequest: TokenExchangeParameters, codeResponse: CodeResponse, cachedAccount: Account, redirectUri: string, cryptoImpl: ICrypto, refreshToken?: string) {
         super(clientId, redirectUri, cryptoImpl);
         this.tokenRequest = tokenRequest;
         this.codeResponse = codeResponse;
+        this.refreshToken = refreshToken || "";
```

Comment:
Any particular reason to default to empty string?

Reply:
Probably not needed, meant to remove this.